plug-ins/Apple.Core/Apple.Core_Unity/Assets/Apple.Core/Editor/AppleBuild.cs

```diff
@@ -382,33 +382,33 @@ private static string GenerateEmbedNativeLibraryShellScript(string projectRelati
             + "dstFrameworkFolder=\"$BUILT_PRODUCTS_DIR/$FRAMEWORKS_FOLDER_PATH\"\n"
             + "dstBundleFolder=\"$BUILT_PRODUCTS_DIR/$PLUGINS_FOLDER_PATH\"\n"
             + $"APPLE_PLUGIN_LIBRARY_ROOT=\"$PROJECT_DIR/{projectRelativeNativeLibraryRoot}\"\n"
-            + "if [ -d $APPLE_PLUGIN_LIBRARY_ROOT ]; then\n"
+            + "if [ -d \"$APPLE_PLUGIN_LIBRARY_ROOT\" ]; then\n"
             + "    for folder in \"$APPLE_PLUGIN_LIBRARY_ROOT\"/*; do\n"
             + "        if [ -d \"$folder\" ]; then\n"
             + "            for item in \"$folder\"/*; do\n"
             + "                if [[ $item = *'.dSYM' ]]; then\n"
             + "                    continue\n"
             + "                elif [[ $item = *'.framework' ]]; then\n"
-            + "                    filename=$(basename $item)\n"
+            + "                    filename=$(basename \"$item\")\n"
             + "                    echo \"    Embedding Apple plug-in framework $filename\"\n"
             + "                    echo \"      Source: $item\"\n"
             + "                    echo \"      Destination: $dstFrameworkFolder/$filename\"\n"
             + "                    if [ ! -z \"$EXPANDED_CODE_SIGN_IDENTITY\" ]; then\n"
             + "                        echo \"      Code signing identity: $EXPANDED_CODE_SIGN_IDENTITY\"\n"
             + "                        codesign --force --sign $EXPANDED_CODE_SIGN_IDENTITY --timestamp\\=none --generate-entitlement-der $item\n"
```

Comment:
@Xaverix this line (and others) also reference `$item` directly, could that cause problems too? If so, can you please also fix this and the other lines that reference `$item` without the double quotes??

Reply:
@VMaldecoasago From what i have tested other lines didn't cause any issues. Some of the lines are simple echo (which is already in quotes). I'll change other lines (e.g. ones that check the extension).